apps/fm/style/fm.css

```diff
@@ -8,6 +8,13 @@ html, body {
   font-size: 10px;
 }
 
+/**
+ * 1. No content inside the app
+ *    should be user-selectable.
+ */
+* {
+  -moz-user-select: none; /* 1 */
```

Comment:
The `1` is a reference to the above comment that says `No content inside the app should be use-selectable`. This is a useful way to comment CSS hacks for conditions like this so its understandable to someone viewing the code in the future as to why the property is needed. If there were several CSS declarations that needed explanation, they could subsequently be marked with a `2`, `3`, etc.